docs/web_quickstart.rst

```diff
@@ -196,6 +196,20 @@ First one is *optimized*. You have got the idea.
 Variable Resources
 ^^^^^^^^^^^^^^^^^^
 
+.. versionchanged:: 3.10
+
+    Fixed paths are preferred over variable paths. For example,
+    if you have two routes ``/a/b`` and ``/a/{name}``, then the first
+    route will always be preferred over the second one.
+
+    If there are multiple dynamic paths with the same fixed prefix,
+    they will be resolved in order of registration.
+
+    For example, if you have two dynamic routes that are prefixed
+    with the fixed ``/users`` path such as ``/users/{x}/{y}/z`` and
+    ``/users/{x}/y/z``, the first one will be preferred over the
+    second one.
```

Comment:
Given this is supposed to be a quickstart, and we've got the exact same description in reference, I'd suggest dropping this. (I think I'd like to take a look at dropping most of this quickstart at some point, and merging it with the tutorial).

Reply:
removed in 2bac4583ee22a55a7ea2a7f3316fe9a8746952e1